composer.json

```diff
@@ -30,7 +30,8 @@
         "phan/phan": "^3.0.2",
         "phpunit/phpunit": "^9.2",
         "yiisoft/composer-config-plugin": "^1.0@dev",
-        "yiisoft/di": "^3.0@dev"
+        "yiisoft/di": "^3.0@dev",
+        "foxy/foxy": "^1.0.8"
```

Comment:
Can't we test aliases without getting packages from npm?

Reply:
It is just a dependency in development, since I added the tests for the use of SASS cli which I think is fine.

Reply:
Let's avoid any external dependencies (especially npm + nodejs).
You can create bash (or just use `cp` command) or php files if you want to test functionality.

Reply:
@terabytesoftw WDYT?

Reply:
The way to work with yii3 npm packages is that, because I must create something different the purpose of the tests is to demonstrate how a package works in reality.

Reply:
I'm not agree with you.
The purpose of the tests is demonstrate that needed command is handled.

Reply:
https://github.com/yiisoft/assets/issues/31